### PR TITLE
Force english lang for selenium

### LINKED ISSUE #24015: [Blazor] Make tests culture independent
Some people with other machine cultures have had trouble running our E2E tests. We should be able to set the culture on the server and likely on the browser too with Selenium with `--lang=en`

### PR DESCRIPTION
**Pull request by HaoK**
Fixes https://github.com/dotnet/aspnetcore/issues/24015

### CODE REVIEW

File: src/Shared/E2ETesting/BrowserFixture.cs [hunk 1]
```diff
@@ -137,6 +137,9 @@ private async Task<(IWebDriver browser, ILogs log)> CreateBrowserAsync(string co
         {
             var opts = new ChromeOptions();
 
+            // Force language to english for tests
+            opts.AddUserProfilePreference("intl.accept_languages", "en");
```
Comment (pranavkm) [inline src/Shared/E2ETesting/BrowserFixture.cs:141]: At least for the BasicTestApp, we force the app to be en-us by default. For server app, the browser language might not make a difference (I don't think we register the localization middleware). Should we modify the apps to configure a default culture instead?

Comment (HaoK) [reply src/Shared/E2ETesting/BrowserFixture.cs:141]: Sure, 

I notice we do have some localization in https://github.com/dotnet/aspnetcore/blob/haok/blazor/src/Components/test/testassets/TestServer/InternationalizationStartup.cs

Should I go ahead and add it to all the various startups in TestServer only, or should I add it to all Startups under test assets?

Also so we don't want to also set it in the ChromeOptions for redundancy?

**Comment by HaoK** [comment]
@pranavkm is this more along the lines of what you had in mind?

**Review by pranavkm — COMMENTED** [review]
Yup. Consider also setting: https://github.com/dotnet/aspnetcore/blob/master/src/Components/test/testassets/BasicTestApp/BasicTestApp.csproj#L13 in the project file. This makes it so that if you run the tests in a non-EN locale, we don't complain about you trying to change the culture as part of running the app.

**Review by pranavkm — APPROVED** [review]

**Pull request merged**